build/WinObjC.Frameworks/WinObjC.Frameworks.nuproj

```diff
@@ -90,7 +96,6 @@
     <ProjectReference Include="$(MSBuildThisFileDirectory)..\MediaPlayer\dll\MediaPlayer.vcxproj" />
     <ProjectReference Include="$(MSBuildThisFileDirectory)..\MessageUI\dll\MessageUI.vcxproj" />
     <ProjectReference Include="$(MSBuildThisFileDirectory)..\Metal\dll\Metal.vcxproj" />
-    <ProjectReference Include="$(MSBuildThisFileDirectory)..\MobileCoreServices\dll\MobileCoreServices.vcxproj" />
     <ProjectReference Include="$(MSBuildThisFileDirectory)..\OpenGLES\dll\OpenGLES.vcxproj" />
```

Comment:
Its moved to be part of the Core package now as Foundation apparently depends on it.

---
In reply to: [102296613](https://github.com/Microsoft/WinObjC/pull/1971#discussion_r102296613) [](ancestors = 102296613)